concepticondata/conceptlists/Girard-1971-546.tsv

```diff
@@ -0,0 +1,547 @@
+ID	NUMBER	SPANISH	GLOSS	CONCEPTICON_ID	CONCEPTICON_GLOSS	SOURCE_ID
```

Comment:
Similar to the comment on the other list, SOURCE_ID should be NUMBER_IN_SOURCE, so we keep some uniformity

Reply:
@AnnikaTjuka I think I tagged you on this already, but since it's a new review I'll add it here: Do we enforce translation for all concepts?

Reply:
As far as I know, we usually have an English translation and @MuffinLinwist is correct that the `concepticon test` requires an ENGLISH or GLOSS column. I re-read the [documentation](https://github.com/concepticon/concepticon-data/blob/master/CONTRIBUTING.md) and the GLOSS column is reserved for English translations so copy-pasting the Spanish glosses is not possible. It's not explicitly mentioned that we enforce a translation, but given the [documentation](https://github.com/concepticon/concepticon-data/blob/master/CONTRIBUTING.md), I'd suggest adding a column GLOSS with English translations. @LinguList Would you agree?

Reply:
I'll leave this question as the only one left in order to merge the PR. The other requested changes are already addressed in the last commit.

Reply:
@LinguList, this is the comment in question.

Reply:
I agree. Gloss is translation to English.

Reply:
Ok. I'll change the concepts on the GLOSS column with translations taken from the CONCEPTICON_GLOSS column.